Tomlet/TomlSerializationMethods.cs

```diff
@@ -75,8 +75,10 @@ static TomlSerializationMethods()
             Register(lt => new TomlLocalTime(lt), value => (value as TomlLocalTime)?.Value ?? throw new TomlTypeMismatchException(typeof(TomlLocalTime), value.GetType(), typeof(TimeSpan)));
         }
 
-        internal static Serialize<object> GetSerializer(Type t)
+        internal static Serialize<object> GetSerializer(Type t, TomlSerializerOptions? options)
         {
+            options ??= new TomlSerializerOptions();
```

Comment:
Given that this is a fairly often-called method, can you make a static TomlSerializerOptions.Default and use that instead here, to avoid an allocation.